spec/definitions/TokenInfoV1.yaml

```diff
@@ -0,0 +1,53 @@
+required:
+  - id
+  - enterprise
+type: object
+properties:
+  id:
+    title: Id
+    type: string
+    format: integer
+    readOnly: true
+  enterprise:
+    title: Enterprise
+    type: string
+    format : uuid
+    readOnly: true
+  user:
+    title : User id
+    type : string
+    readOnly : true
+  developerapp:
+    title : Developer app
+    type : string
+    format : uuid
+    readOnly : true
+  source_refresh_token:
```

Comment:
We need a description of what this and can be used for ?

Reply:
done